### PR TITLE
WSL file access from Windows

### LINKED ISSUE #34: WSL file access from Windows
The current tutorials advise WSL users to put files in `/mnt/c/...` so that it's easier to access them from Windows.  We should instead recommend putting the files in the Linux home and using the recently-added [File Explorer Linux shortcut](https://devblogs.microsoft.com/commandline/access-linux-filesystems-in-windows-and-wsl-2/#accessing-these-files-with-file-explorer).

### PR DESCRIPTION
**Pull request by jsliu67**
Explain where WSL stores files and how to access them from the Windows host.

Closes #34

### CODE REVIEW

File: docs/setup_vscode.md [hunk 1]
```diff
@@ -155,7 +158,32 @@ $ code .
 ```
 </div>
 
-<img src="images/vscode020.png" width="768px" />
+<img src="images/vscode020.png" width="512px" />
+
+### Windows
```
Comment (awdeorio) [inline docs/setup_vscode.md:163]: This information might also be needed by Visual Studio users.  Would it make sense to put the detailed description with the screenshot in `setup_wsl.md` and then link to it here with a pro-tip?

**Comment by awdeorio** [comment]
Thanks for getting this posted so quickly @jsliu67 !

File: docs/setup_vscode.md [hunk 2]
```diff
@@ -155,7 +158,32 @@ $ code .
...
+To find Linux files in the file explorer, navigate to `/home`, and open the file explorer there using the command line.
+
+```console
+$ cd /home
+$ explorer.exe .
+```
+
+You'll see your home folder. Right click it and choose "Pin to Quick access". Now you'll have a shortcut to your linux home directory. 
+
+<img src="images/vscode022.png" width="768px" />
```
Comment (awdeorio) [inline docs/setup_vscode.md:186]: Could you make a second screenshot that shows the little linux icon in the lower left of the explorer?

File: docs/setup_vscode.md (hunk 1 above)
Comment (jsliu67) [reply docs/setup_vscode.md:163]: Seems like a good idea, I'll add it in

File: docs/setup_vscode.md [hunk 3]
```diff
@@ -126,10 +126,13 @@ To create a VS Code project, create a folder (directory).  There are many ways t
 
 **Windows/WSL:** Open the Ubuntu application.
 
+Your root directory is located at [tilde `~`](cli.html#home-directory-). We recommend that you store your project files here.
```
Comment (ohjuny) [inline docs/setup_vscode.md:129]:
```suggestion
Your home directory is located at [tilde `~`](cli.html#home-directory-). We recommend that you store your project files here.
```

File: docs/setup_vscode.md [hunk 4]
```diff
@@ -143,7 +146,7 @@ $ cd p1-stats
 | `Project 1 Stats/` | `p1-stats/` |
 
 </div>
-
+
```
Comment (ohjuny) [inline docs/setup_vscode.md:149]:
```suggestion

```

File: docs/setup_wsl.md [hunk 5]
```diff
@@ -118,11 +118,30 @@ Enable "Use Ctrl+Shift+C/V Copy/Paste" option in the Console "Options" propertie
 
 <img src="https://devblogs.microsoft.com/wp-content/uploads/sites/33/2019/04/copy-paste.png" width=480px>
 
-### Accessing Windows files
-Windows files are accessible from Linux at `/mnt/c/`.  You have a separate Ubuntu home directory, e.g., `/home/awdeorio/`.
+### Accessing files with Linux
```
Comment (ohjuny) [inline docs/setup_wsl.md:121]:
```suggestion
### Accessing files with WSL
```

**Review by ohjuny — COMMENTED** [review]
Made a couple small suggestions, in general looks good!

**Comment by awdeorio** [comment]
I pushed some changes, take a look @ohjuny and @jsliu67 .  Also, @jsliu67 would you please fill in the "FIXME" screenshots?  I don't have a windows machine.  Thanks!

File: docs/setup_vscode.md (hunk 4 above)
Comment (awdeorio) [reply docs/setup_vscode.md:147]: This is a good reminder to check the diff on your own PRs.  An ideal PR has the changes you need for the new feature, but not any other distracting changes.

**Review by awdeorio — APPROVED** [review]
LGTM!

**Pull request merged**